app/views/file/_line_sidebar.html.erb

```diff
@@ -3,16 +3,12 @@
 <% object = version.inv_object %>
 <li style="word-wrap:break-word;">
   <% if can_download %>
-    <% if file.exceeds_download_size? %>
-      <%= File.basename(file.pathname) %> (too large to download)
-    <% else %>
-      <%= link_to File.basename(file.pathname),
-          :controller       => :file,
-          :action           => :download,
-          :object           => object,
-          :version          => version,
-          :file             => file %>
-    <% end %>
+    <%= link_to File.basename(file.pathname),
+      :controller       => :file,
```

Comment:
try not to mix use of `{ key: "value" }` (used above in download_response for example) and the older `{ :key => "value" }` style